Regression - can't do a graceful shutdown

**What happened**:

Clean shutdowns of the server no longer work when doing `Ctrl+C` in the terminal.  I suspect the parallel Bull worker processes are holding open the main process.

**What should have happened**:

It should allow a graceful shutdown.

**How to reproduce it (as precise as possible)**:

In your `.env` set `FEED_QUEUE_PARALLEL_WORKERS=*` so it will run multiple parallel workers.  Then start the server with `npm start`.  Once it's going, try doing `CTRL+C`

```
[ 2020-01-25 17:53:50.229 ] INFO  (64137 on brightness.local): Received SIGINT, starting shut down
[ 2020-01-25 17:53:50.232 ] ERROR (64137 on brightness.local):
    error: "SIGINT"
[ 2020-01-25 17:53:50.249 ] ERROR (64137 on brightness.local): Job 4660 failed.
    error: {}
[ 2020-01-25 17:53:50.250 ] ERROR (64137 on brightness.local): Job 3064 failed.
    error: {}
[ 2020-01-25 17:53:50.250 ] ERROR (64137 on brightness.local): Job 4468 failed.
    error: {}
[ 2020-01-25 17:53:50.254 ] ERROR (64137 on brightness.local): Job 4648 failed.
    error: {}
[ 2020-01-25 17:53:50.255 ] INFO  (64137 on brightness.local): Feed queue shut down.
[ 2020-01-25 17:54:00.238 ] ERROR (64137 on brightness.local): Could not close connections in time, forcefully shutting down
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! telescope@0.0.1 start: `node src/backend`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the telescope@0.0.1 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/humphd/.npm/_logs/2020-01-25T22_54_00_264Z-debug.log
```

**Environment**:

- OS: macOS

--- a/package.json
+++ b/package.json
@@ -83,6 +83,7 @@
     "sanitize-html": "1.20.1",
     "sentiment": "5.0.2",
     "spam-filter": "1.1.1",
+    "stoppable": "1.1.0",
     "valid-url": "1.0.9",
     "wordcount": "1.1.1"
   },

--- a/src/backend/lib/shutdown.js
+++ b/src/backend/lib/shutdown.js
@@ -16,7 +16,10 @@ async function stopQueue() {
 }
 
 async function stopWebServer() {
-  const serverClose = promisify(server.close.bind(server));
+  // Use stopppable's server.stop() instead of httpServer.close()
+  // to force connections to close as well.  See:
+  // https://github.com/hunterloftis/stoppable
+  const serverClose = promisify(server.stop.bind(server));
   try {
     await serverClose();
     logger.info('Web server shut down.');

--- a/src/backend/web/server.js
+++ b/src/backend/web/server.js
@@ -1,11 +1,16 @@
 require('../lib/config');
+const stoppable = require('stoppable');
+
 const app = require('./app.js');
 
 const { logger } = app.get('logger');
 const HTTP_PORT = process.env.PORT || 3000;
 
-const server = app.listen(HTTP_PORT, () => {
-  logger.info(`Telescope listening on port ${HTTP_PORT}`);
-});
+// Make our server not block when we want it to close.
+const server = stoppable(
+  app.listen(HTTP_PORT, () => {
+    logger.info(`Telescope listening on port ${HTTP_PORT}`);
+  })
+);
 
 module.exports = server;

